config/webpack/loaders/devSitePackage.js

```diff
@@ -12,7 +12,7 @@ const loader = async function loader(content) {
   let finalUrl = '';
   if (repoUrl) {
     if (repoDirectory) {
-      finalUrl = `${repoUrl}/tree/master/${repoDirectory}`;
+      finalUrl = `${repoUrl}/tree/main/${repoDirectory}`;
```

Comment:
This will have implications until all repos are changed to main.... no way around it.